src/libraries/Common/tests/System/Net/WebSockets/WebSocketStream.cs

```diff
@@ -177,7 +177,7 @@ protected override void Dispose(bool disposing)
         public async override Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken)
         {
             ValidateBufferArguments(buffer, offset, count);
-            ThrowIfDisposed();
+            ObjectDisposedException.ThrowIf(_disposed != 0, this);
```

Comment:
For these cases where there's an existing `ThrowIfDisposed()` parameterless method, I would still keep it.  Otherwise, as the changes in this file highlight, we end up duplicating the disposal condition in a bunch of places.